### DISCUSSION TITLE
Deploy freezes on "Building fresh packages..."

### DISCUSSION THREAD

**Discussion by ivomirtolev** [vercel/vercel#8153] [Help]
Hi all,

I have a strange problem which appeared in the deploy of an angular project. I'm using yarn package alias for a dependency. As an example: 

    "@package": "1.9.5",
    "@package-1.9.2": "npm:@package@1.9.2",
    "@package-1.8.19": "npm:@package@1.8.19"
    
 The deployment was working fine, but three days ago the deployment started to freeze on the step Building fresh packages... and timeouts after 45 minutes. My build and development settings are the default for an Angular project. I have debugged and found that if i remove the second (1.9.2) or the third (1.8.19) dependency, the deployment works again. 
 
 There are no changes in the dependencies. I have also tried to redeploy an old commit containing the three packages and it is also not working, although it was working before two weeks.
 
I have another angular project with the same Vercel settings, also using these dependencies and everything works as expected there. I have also tried to create a new project in Vercel but the freeze appears again.

 Any ideas what can be the problem ?

**Comment by amyegan** [comment]
Hi @ivomirtolev. Can you share your repo or a minimal reproduction of the problem?
